builtin/providers/aws/resource_aws_elb_load_balancer_listener_policy.go

```diff
@@ -0,0 +1,139 @@
+package aws
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsLoadBalancerListenerPolicies() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLoadBalancerListenerPoliciesCreate,
+		Read:   resourceAwsLoadBalancerListenerPoliciesRead,
+		Update: resourceAwsLoadBalancerListenerPoliciesCreate,
+		Delete: resourceAwsLoadBalancerListenerPoliciesDelete,
+
+		Schema: map[string]*schema.Schema{
+			"load_balancer_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"policy_names": &schema.Schema{
+				Type:     schema.TypeSet,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Optional: true,
+				Set:      schema.HashString,
+			},
+
+			"load_balancer_port": &schema.Schema{
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsLoadBalancerListenerPoliciesCreate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	loadBalancerName := d.Get("load_balancer_name")
+
+	policyNames := []*string{}
+	if v, ok := d.GetOk("policy_names"); ok {
+		policyNames = expandStringList(v.(*schema.Set).List())
+	}
+
+	setOpts := &elb.SetLoadBalancerPoliciesOfListenerInput{
+		LoadBalancerName: aws.String(loadBalancerName.(string)),
+		LoadBalancerPort: aws.Int64(int64(d.Get("load_balancer_port").(int))),
+		PolicyNames:      policyNames,
+	}
+
+	if _, err := elbconn.SetLoadBalancerPoliciesOfListener(setOpts); err != nil {
+		return fmt.Errorf("Error setting LoadBalancerPoliciesOfListener: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s:%s", *setOpts.LoadBalancerName, strconv.FormatInt(*setOpts.LoadBalancerPort, 10)))
+	d.Set("load_balancer_name", setOpts.LoadBalancerName)
```

Comment:
addressed in https://github.com/hashicorp/terraform/pull/7458/commits/4f07b924479759b47ab6df4baa906bbee5b75371